src/Microsoft.TemplateEngine.Abstractions/IFileChange.cs

```diff
@@ -24,6 +26,7 @@ public interface IFileChange
         /// <summary>
         /// Gets the file content.
         /// </summary>
+        [Obsolete("file contents are not returned in this version, the property always empty byte array.")]
```

Comment:
A bit of cleaning up:
```suggestion
        [Obsolete("File contents are no longer available. This property always returns an empty byte array.")]
```